EMS/core-bundle/src/Controller/ContentManagement/JobController.php

```diff
@@ -60,12 +61,12 @@ public function index(Request $request): Response
         return $this->render("@$this->templateNamespace/crud/overview.html.twig", [
             'form' => $form->createView(),
             'icon' => 'fa fa-file-text-o',
-            'title' => t('type.title_overview', ['type' => 'job'], 'emsco-core'),
```

Comment:
no you can not use php constants inside the t() function. The parser will search all php code, but will not resolve constants. so we need to keep the string 'emsco-core'